docs/infrastructure/native-image.md

```diff
@@ -117,7 +117,20 @@ It is possible that different classes are reflectively accessed on different
 platforms. In that case it may be necessary to run the agent on multiple
 platforms and merge the configs. If the conflicts were conflicting (i.e. some
 reflectively accessed classes existed only on one platform), it may be necessary
-to maintain separate configs for each platform. Currently in the Launcher this
-is not the case - the reflective accesses seem to be platform independent, as
-the launcher built with a config created on Linux runs successfully on other
-platforms.
+to maintain separate configs for each platform. Currently, the
+`native-image-agent` is not available on Windows, so Windows-specific reflective
+accesses may have to be gathered manually. For some types of accesses it may be
+possible to force the Windows-specific code paths to run on Linux and gather
+these accesses semi-automatically. That is currently the case with
+archive-related accesses - while the Linux launcher never unpacks ZIP files, we
+can manually force it to do so, to register the reflection configuration that
+will than be used on Windows to enable ZIP extraction.
+
+Moreover, some reflective accesses may not be detected by the tool
+automatically, so they may need to be added manually. One of them is an access
+to the class `[B` when using Akka, so it would require manually adding it to the
```

Comment:
And yet it is required to be able to run Akka :sweat_smile: 

It may be related to https://stackoverflow.com/a/17670771, or more generally Akka somewhere tries to reflectively instantiate (?) an array of bytes (whose `.class.getName()` is `[B`) and so without this magical line it fails with a `ClassNotFoundException: [B`.

Adding this to the relfection configuration makes Akka work, but it does feel like magic. If necessary I can try digging into what exactly happens here to better understand this.

Reply:
Huh, as long as it's needed don't bother digging. I'd suggest explaining the docs some more though.